config/dev

```diff
@@ -88,15 +88,18 @@ DARK_CONFIG_ROLLBAR_POST_CLIENT_ITEM=c7af77e991aa4edd80cf6a576c1e42f5
 DARK_CONFIG_ROLLBAR_POST_SERVER_ITEM=e2777a52d9a24c00a7e8e8aedaa61017
 
 # Honeycomb
-DARK_CONFIG_TELEMETRY_EXPORTER=console
-DARK_CONFIG_HONEYCOMB_API_KEY=none
+DARK_CONFIG_TELEMETRY_EXPORTER=honeycomb
+DARK_CONFIG_HONEYCOMB_API_KEY=76f291cb893c40ed06f8f91620b5ee3c
```

Comment:
This is safe to push? If this is some non-prod public key, do we need to worry about over-usage at all?

Reply:
Sigh. No, this was a mistake. I've disabled the key